scripts/install-test-fixture.sh

```diff
@@ -1,5 +1,12 @@
-TEST_FIXTURE_SCOPES="--scope @coolwallet/core --scope @coolwallet/testing-library --scope @coolwallet/transport-jre-http"
+#!/bin/bash
+
+TEST_FIXTURE_SCOPES=(
+  --scope @coolwallet/core
+  --scope @coolwallet/testing-library
+  --scope @coolwallet/transport-jre-http
+)
+
 LERNA="npx lerna"
 
-$LERNA bootstrap $TEST_FIXTURE_SCOPES
-$LERNA run build $TEST_FIXTURE_SCOPES
```

Comment:
Convenient for local App testing. This script will generate libs and delete node_modules. It will make App to install faster while referring from local files.